graphql/src/index.ts

```diff
@@ -199,6 +199,8 @@ const combinedSchema = makeExecutableSchema({
         elasticSearchAPI: new ElasticSearchAPI(),
       };
     },
+    introspection: true,
+    playground: process.env.NODE_ENV === "development",
```

Comment:
AFAIK, you wanted to leave the playground enabled for staging?

Usually the staging environment has a production build, meaning that `NODE_ENV` is production. E.g. https://github.com/City-of-Helsinki/unified-search/blob/develop/graphql/Dockerfile#L75

Reply:
Right, I'll use separate env variable for this purpose instead of relying NODE_ENV.